chart/templates/nginx.yaml

```diff
@@ -35,6 +33,17 @@ data:
             proxy_pass $selected_upstream;
             ssl_preread on;
         }
+
+        {{- range .Values.extraPorts }}
+        upstream backend_tenant_{{ . }} {
+            server ###ZARF_VAR_BASE_IP###.201:{{ . }}; 
+        }
+
+        server {
+            listen {{ . }};
+            proxy_pass backend_tenant_{{ . }};
+        }
+        {{- end }}
```

Comment:
I may just be missing the connection here, but how would you intend to use `backend_tenant_22` for example? No hosts would point to it, so while it would be exposed I'm not sure it would be routable via a hostname?

Let me know if I'm just missing the wiring here.

Reply:
https://github.com/defenseunicorns/uds-package-gitlab/pull/196/files#diff-17ed18489a956f326ec0fe4040850c5bc9261d4631fb42da4c52891d74a59180R76
https://github.com/defenseunicorns/uds-package-gitlab/pull/196/files#diff-8cb1344ac0843ca80af6191a6034a36e5cea2f6d9e8f65a734e282210a8892c6

Those are the docs and the ingress wirings from the GitLab PR (stolen from here: https://github.com/defenseunicorns/uds-bundle-software-factory-nutanix/tree/main/packages/additional-manifests/gitlab)